libnavigation-ui/src/main/res/values/styles.xml

```diff
@@ -26,6 +26,7 @@
     </style>
 
     <style name="NavigationViewLight" parent="Theme.MaterialComponents.Light.NoActionBar">
+        <item name="android:colorPrimary" tools:targetApi="lollipop">@android:color/white</item>
```

Comment:
We support `minSdk 19` could targeting Lollipop only here be an issue?

Reply:
no, verified on Android 19 emulator and it doesn't have side effect. and it works well.

Reply:
🆗 Thanks for confirming. I've just noticed that this is `tools` 🤦 i.e. only for the IDE so I believe we can remove. Why do we need it 🤔?